### PR TITLE
feat(deployment-row): add link to history logs

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-781)

- Add link to deployment history for deployments rows
- Plan for next sprint but quick win for users
- ButtonIcon should be clean (UI like button-action component)

Capture d’écran 2023-08-11 à 11 13 14

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> One day they will all understand

File: libs/shared/ui/src/lib/components/table/table-row-deployment/table-row-deployment.spec.tsx [hunk 1]
```diff
@@ -1,13 +1,24 @@
-import { render } from '__tests__/utils/setup-jest'
+import { render, screen } from '__tests__/utils/setup-jest'
+import { ServiceTypeEnum, StateEnum } from 'qovery-typescript-axios'
 // eslint-disable-next-line @nx/enforce-module-boundaries
 import { applicationDeploymentsFactoryMock } from '@qovery/shared/factories'
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import { renderWithProviders } from '@qovery/shared/util-tests'
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/table/table-row-deployment/table-row-deployment.spec.tsx:6]: Usage of util-tests on shared UI library causes en circular boundaries

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] staging - feat(deployment-row): add link to history logs - 2023-08-11T13:12:26Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/e7700438-f919-4b1a-b6fd-9668f09e8ca9/services/general)

 Another comment will be posted when deployments are terminated

File: libs/shared/ui/src/lib/components/table/table-row-deployment/table-row-deployment.tsx [hunk 2]
```diff
@@ -149,9 +145,22 @@ export function TableRowDeployment(props: TableRowDeploymentProps) {
...
+                size={ButtonSize.SMALL}
+                onClick={() =>
+                  navigate(
+                    fromService
+                      ? pathEnvironmentLogs + SERVICE_LOGS_URL(data?.id)
+                      : pathEnvironmentLogs +
+                          DEPLOYMENT_LOGS_VERSION_URL(data?.id, (data as DeploymentService).execution_id)
+                  )
+                }
+                className="!w-7 !h-7 !border-r btn-icon-action__element"
```
Comment (ctjhoa) [inline libs/shared/ui/src/lib/components/table/table-row-deployment/table-row-deployment.tsx:161]: Add important is a bad practice. We must find another way to style stuff without overriding existing rules

**Pull request merged**